js/data/bucket.js

```diff
@@ -81,7 +81,7 @@ function Bucket(options) {
  * @private
  */
 Bucket.prototype.addFeatures = function() {
-    for (var i = 0; i < this.features.length; i++) {
+    for (var i = this.features.length - 1; i >= 0; i--) {
```

Comment:
Because we are using the `gl.ONE_MINUS_DST_ALPHA` blending function, features are effectively rendered in reverse order.